native-file-system/idlharness.https.any.js

```diff
@@ -0,0 +1,15 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+idl_test(
+  ['native-file-system'],
+  ['html', 'dom'],
+  idl_array => {
+    idl_array.add_objects({
+      // TODO: Add instances of FileSystemHandle, FileSystemFileHandle,
```

Comment:
Is there a reason these can't be done as part of this PR?

Reply:
I didn’t spend any time checking, but assume that access is gated on some kind of permission prompt. But maybe the permission can be set and make it work, I’ll check before merging this.

Reply:
I took a look and filed https://github.com/WICG/native-file-system/issues/150. One could also make this test manual or create a second manual one, but that doesn't really seem worth it since there's no use of `[Unforgeable]` or other things that require an instance to check.